api/services/ocean/gke/paths/clusterFetchElastilog.yaml

```diff
@@ -2,8 +2,10 @@ summary: Ocean for GKE
 get:
   summary: "Fetch Elastilog"
   description: >
-    Fetch an Ocean Cluster's log
-  operationId: "oceanGkeFetchLog"
+    Roll an entire Ocean cluster or roll subsets of the cluster by specifying launchSpecIds or instanceIds. Performing this request will start the roll immediately, by defined batches.
```

Comment:
@vlad666spot I think this is a wrong description for a fetch log.
http://help.dev.spotinst.com/spotinst-api/ocean/ocean-cloud-api/ocean-for-gke/fetch-elastilog/

Reply:
Fixed